src/openforms/tests/test_frontend_utils.py

```diff
@@ -0,0 +1,52 @@
+from django.test import TestCase
+
+from openforms.frontend import get_frontend_redirect_url
+from openforms.submissions.tests.factories import SubmissionFactory
+
+
+class FrontendRedirectTests(TestCase):
```

Comment:
Nice tests! Can you add a case with a querystring parameter in `Submission.form_url`?